src/Calypso-Browser/ClyQueryViewMorph.class.st

```diff
@@ -677,6 +700,23 @@ ClyQueryViewMorph >> setUpDataSource: aDataSource [
 	self ensureSelectedItemIfNeeded
 ]
 
+{ #category : #'drag and drop support' }
+ClyQueryViewMorph >> shouldExpand: aSelectedItem [
```

Comment:
maybe #isReadyToExpandForDrop:

Reply:
That's better.
My problem is that I'd like to show that this is indeed a tester, but one that modifies the state